### PR TITLE
DOP-1006: Fix version selector and title in dropdown

### PR DESCRIPTION
**Pull request by sophstad**
[DOP-1006] [[BI Connector Staging](https://docs-mongodbcom-staging.corp.mongodb.com/snooty/bi-connector/sophstad/DOP-1006/)] [[Manual Staging](https://docs-mongodbcom-staging.corp.mongodb.com/test2/docs/sophstad/DOP-1006/)] Fix title and version selector in the sidebar so that only the table of contents scrolls.

### CODE REVIEW

**Review by allisonmui9 — APPROVED** [review]
Looks perfect. Thanks Sophie!

**Comment by mango-db** [comment]
Looks like a unit test is failing: `'TOC exists with proper number of sections and nodes per section'`.

**Review by mango-db — COMMENTED** [review]
BI-connector staging looks fine, but I'm unable to scroll down in the left nav to fully view "Technical Support" .

image

When I open the "Reference" drawer, I can see even less of the leftnav content.

image

Is this the intended effect?

File: tests/unit/TableOfContents.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
```
Comment (mango-db) [inline tests/unit/TableOfContents.test.js:2]: What is the reason behind this change?

Comment (sophstad) [reply tests/unit/TableOfContents.test.js:2]: Shallowly rendering the component did not properly allow [jest-emotion](https://www.npmjs.com/package/jest-emotion) to resolve the `css` prop. Using `mount` instead ensures consistent output for this prop.

**Review by mango-db — APPROVED** [review]
LGTM!
